sofa-ark-parent/core-impl/container/src/test/java/com/alipay/sofa/ark/container/service/api/ArkClientTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.ark.container.service.api;
+
+import com.alipay.sofa.ark.api.ArkClient;
+import com.alipay.sofa.ark.api.ClientResponse;
+import com.alipay.sofa.ark.api.ResponseCode;
+import com.alipay.sofa.ark.container.BaseTest;
+import com.alipay.sofa.ark.spi.model.BizInfo;
+import com.alipay.sofa.ark.spi.model.BizState;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URL;
+
+/**
+ * @author qilong.zql
+ * @since 0.6.0
+ */
+public class ArkClientTest extends BaseTest {
+
+    // bizName=biz-demo, bizVersion=1.0.0
+    private URL bizUrl1;
+    // bizName=biz-demo, bizVersion=2.0.0
+    private URL bizUrl2;
+
+    @Before
+    public void before() {
+        super.before();
+        // bizName=biz-demo, bizVersion=1.0.0
+        bizUrl1 = this.getClass().getClassLoader().getResource("sample-ark-1.0.0-ark-biz.jar");
+        // bizName=biz-demo, bizVersion=2.0.0
+        bizUrl2 = this.getClass().getClassLoader().getResource("sample-ark-2.0.0-ark-biz.jar");
+    }
+
+    @Test
+    public void testOperationBiz() throws Throwable {
```

Comment:
It's recommended to test only one checkpoint on every case, this case is a bit long